components/avatar/avatar.css

```diff
@@ -0,0 +1,18 @@
+.avatar {
+  height: 6rem;
```

Comment:
I would suggest to add width as well, though it calculates based on height it is a good practice to give width and height for image elements